stable/pgadmin/Chart.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: pgAdmin is a web based administration tool for PostgreSQL database
+name: pgadmin
+version: 0.1.1
```

should be 
```suggestion
version: 1.0.0
```
since it is in the stable folder and will be the first release